aldryn_client/localdev/main.py

```diff
@@ -0,0 +1,85 @@
+import os
+import subprocess
+
+import click
+
+
+def create_workspace(client, website_slug, path=None):
+    website_git_path = 'git@github.com:aldryn/test-project.git'  # temporary
+
+    path = os.path.abspath(path) if path else None
+    website_id = client.get_website_id_for_slug(website_slug)
+
+    docker_compose_base = [
+        'docker-compose', '-f', os.path.join(path, 'docker-compose.yml')
+    ]
+
+    def docker_compose(cmd):
+        if not isinstance(cmd, list):
+            cmd = [cmd]
+        return docker_compose_base + cmd
+
+    try:
+        # clone project
+        clone_args = ['git', 'clone', website_git_path]
+        if path:
+            clone_args.append(path)
+        subprocess.check_output(clone_args)
+
+    except subprocess.CalledProcessError as exc:
+        raise click.ClickException(exc.output)
+
+    # create postgres data container
+    dbdata_container = 'aldryn_{}_dbdata'.format(website_id)
+
+    existing_data_container_id = subprocess.check_output([
+        'docker', 'ps', '-a', '-q',
+        '--filter=name={}'.format(dbdata_container),
+    ]).replace(os.linesep, '')
+
+    reuse_data_container = False
+    if existing_data_container_id:
+        reuse_data_container = click.confirm(
+            'There\'s an existing database container for this project '
```

Comment:
Theoretically we should use double quotes if we're quoting single quotes... (personally I prefer this style as well, but PEP is PEP :P)